prisma-fmt/src/main.rs

```diff
@@ -1,6 +1,6 @@
 mod actions;
 mod format;
-mod lint;
+// mod lint;
```

Comment:
Changes to `main.rs` are necessary to let `prisma-fmt` compile.
I commented out the `prisma-fmt lint` command keeping in mind that, with https://github.com/prisma/team-orm/issues/1100, we will soon get rid of `prisma-fmt`'s binary anyway